package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "alias-command",
   "main": "./lib/atom-alias-command",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
<minor nitpick>I usually leave version bumps out of PRs so the package maintainer can use [npm version](https://docs.npmjs.com/cli/version) at release time</minor nitpick>